src/org/loklak/harvester/YoutubeScraper.java

```diff
@@ -357,6 +357,16 @@ public Post parseVideo(final BufferedReader br, String type, String url) {
                         json.put("youtube_description", text);
                         continue;
                     }
+                    if(parse_channel_name) {
```

Comment:
The code looks good. But it would be better if you make this conditional consistent with others - 
```
if(...) {    =>    if (...) {
```

Reply:
Updated the PR